code/modules/uplink/uplink_items.dm

```diff
@@ -1217,6 +1217,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	restricted_roles = list("Mime")
 	surplus = 0
 
+/datum/uplink_item/role_restricted/fluorite
+	name = "Fluorite Octet"
+	desc = "Rolling the dice will execute a wide range of highly unpredictable effects. Infinite uses."
+	cost = 8
```

Comment:
Too expensive for no direct advantage.

Reply:
reminder that people will tc trade for the dice of fate and it's so good that admins won't give you the infinite version